samples/bluetooth/scanning_while_connecting/central/README.rst

```diff
@@ -0,0 +1,238 @@
+.. _bt_scanning_while_connecting:
+
+Bluetooth: Scanning while connecting
+####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to reduce the time to establish connections to many devices, typically done when provisioning devices to a network.
+The total connection establishment time is reduced by scanning while connecting and by using the filter accept list.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires at least one other development kit running a sample advertising with the name set with :kconfig:option:`CONFIG_BT_DEVICE_NAME`.
+It is recommended to use the sample :ref:`peripheral_uart`, because this sample acts as multiple peripheral devices.
+
+Overview
+********
+
+You can use this sample as a starting point to implement an application designed to provision a network of Bluetooth peripherals.
+The approaches demonstrated in this sample will reduce the total time to connect to many devices.
+A typical use-case is a gateway in a network of devices using Periodic Advertising with Responses.
+
+To illustrate how connection establishment speed can be improved, it measures the time needed to connect to 16 devices with three different modes: sequential scanning and connection establishment, concurrent scanning while connecting, and concurrent scanning while connecting with the filter accept list.
+It will connect to any device that matches the device name set with :kconfig:option:`CONFIG_BT_DEVICE_NAME`.
```

Comment:
```suggestion
To illustrate how connection establishment speed can be improved, it measures the time needed to connect to 16 devices with three different modes:

* Sequential scanning and connection establishment
* Concurrent scanning while connecting
* Concurrent scanning while connecting with the filter accept list

It will connect to any device that matches the device name set with the :kconfig:option:`CONFIG_BT_DEVICE_NAME` Kconfig option.
```
It (ín two places) referring here to the sample?